### PR TITLE
feat: add Dockerfile

### PR DESCRIPTION
**Pull request by Armansaqarchi**

### CODE REVIEW

File: output.txt
Comment (farbodbj) [inline output.txt:1]: do not change this file

File: subfinder
Comment (farbodbj) [inline subfinder:1]: add to .gitignore

**Comment by farbodbj** [comment]
@keivanipchihagh review this merge request

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM golang:1.21-alpine AS builder
+
+WORKDIR /go/src
+# Add gcc and libc-dev early so it is cached
+RUN set -xe \
+	&& apk add  gcc libc-dev git curl
```
Comment (keivanipchihagh) [inline Dockerfile:6]: Do we need `gcc` for this?

**Comment by Armansaqarchi** [comment]
You right, i forgot to remove it. it comes from the docker template.

**Pull request merged**